src/util/getInMemoryPage.js

```diff
@@ -0,0 +1,41 @@
+const sorter = id => (a, b) => {
+  var nameA = a[id]
+  var nameB = b[id]
+  if (typeof nameA === 'string') nameA.toUpperCase()
+  if (typeof nameB === 'string') nameB.toUpperCase()
+  if (nameA < nameB) return -1
+  if (nameA > nameB) return 1
+  return 0
+}
+
+const fetchPage = (data, filter) => {
+  const {page = 0, pageSize = 10} = filter
+  const pageData = []
+  const start = page * pageSize
+  for (let i = start; i < (start + pageSize); i++) {
+    if (i >= 0 && i < data.length) {
+      pageData.push(data[i])
+    }
+  }
+  return pageData
+}
+
+const sortData = (data, sorts) => {
```

Comment:
I would like to see a note about, or a TODO about full implementation of sorting here